src/nvim/main.c

```diff
@@ -299,7 +299,7 @@ int main(int argc, char **argv)
   // Don't redraw until much later.
   RedrawingDisabled++;
 
-  setbuf(stdout, NULL);
+  setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
Vim patch switches from full buffering to line buffering but Windows does not have line buffering and the current Neovim code and tests rely on no buffering.